draft-sharabayko-mops-srt.md

```diff
@@ -1100,6 +1103,42 @@ Control Information Field (CIF).
 : A single value or a range of lost packets sequence numbers. See packet sequence number
 coding in {{packet-seq-list-coding}}.
 
+### Congestion Warning {#ctrl-pkt-congestion}
+
+The Congestion Warning control packet is not in use at the moment.
```

Comment:
It is a legacy packet. The sender never sends it. But the receiver has a known behavior upon reception of this packet. Therefore, to avoid potential backward incompatibility this description is beneficial.